README.md

```diff
@@ -289,10 +310,19 @@ page.invokeMethod('evaluate', function(selector) {
 });
 ```
 
+#### `page#invokeMethod('setContent', expectedContent, expectedLocation)`
+
+Set content and/or URL address of the page, without waiting for additional elements to load.
+
+If `page#invokeMethod('render', ...)` is called right after, it might not execute properly (see `page#loadContent` notes).
+
+#### `page#invokeMethod('render', filename, [{format, quality}])`
```

Comment:
I am a little confused to what you are doing here. These are all implemented and don't require `invokeMethod`. See https://github.com/amir20/phantomjs-node/blob/master/src/page.js#L101